.github/workflows/validate.yaml

```diff
@@ -12,18 +13,30 @@ jobs:
         with:
           fetch-depth: 0  # For git diff
 
-      - name: Download the JSON schema
-        run: wget https://raw.githubusercontent.com/ossf/osv-schema/v1.6.7/validation/schema.json
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 'stable'
 
-      - name: Install check-jsonschema
-        run: pip install check-jsonschema
+      - name: Install osv-linter
+        run: |
+          git clone https://github.com/ossf/osv-schema.git
```

Comment:
Should pin the version

```suggestion
          git clone https://github.com/ossf/osv-schema.git -b v1.6.7
```

Reply:
make sense 👍 
pinned in https://github.com/aquasecurity/vuln-list-aqua/pull/2/commits/af4136e8734288ab66aa180f069ff9c29255fa79